torchvision/datasets/video_utils.py

```diff
@@ -213,6 +214,8 @@ def compute_clips_for_video(
         total_frames = len(video_pts) * (float(frame_rate) / fps)
         _idxs = VideoClips._resample_video_idx(int(math.floor(total_frames)), fps, frame_rate)
         video_pts = video_pts[_idxs]
+        if num_frames is None:
+            num_frames = int(total_frames)
```

Comment:
This can lead to videos so short that don't make the minimum temporal dimension of the model, leading to errors such as:
```
RuntimeError: input image (T: 1 H: 7 W: 7) smaller than kernel size (kT: 2 kH: 7 kW: 7)
```